lib/LanguagePicker.js

```diff
@@ -32,10 +29,15 @@ function LanguagePicker(lang = 'en', config = {}) {
   });
 
   // Add known fallbacks for the language
-  fallbackMap = config.languageMap || {};
-  if (fallbackMap[lang]) {
-    fallbacks = Array.isArray(fallbackMap[lang]) ?
-      fallbackMap[lang] : [fallbackMap[lang]];
+  let fallbacks;
+  if (config.languageMap) {
+    fallbacks = config.languageMap[lang];
+    if (fallbacks && !Array.isArray(fallbacks)) {
+      fallbacks = [fallbacks];
+    }
+  }
+  if (!fallbacks) {
+    fallbacks = [];
```

Comment:
(my previous comment was detached from this, weirdly) 

Why is this better than the original, shorter, code? It only runs once, so there's no need to micro-optimize, and it is longer and less clear?